inspire/base/templates/header.html

```diff
@@ -128,14 +104,37 @@
   </nav>
 </section>
 
-{% if request.path  == "/search" %}
+{% if request.path  == "/search" or request.path | is_collection_path %}
```

Comment:
instead of hardcoding the URLs, endpoints should be used.

For example:

```python
{%- if request.endpoint == 'collections.index' %}
```

You can see all the endpoints defined in the application with:

```bash
$ inveniomanage show-urls
```

And the `flask-toolbar` also displays some info about the current view being used

![screen shot 2015-10-30 at 2 58 43 pm](https://cloud.githubusercontent.com/assets/580727/10847438/c5667b2c-7f16-11e5-8e44-253307e1883d.png)